plugins/aws/workspaces/workspacesInstanceCount.js

```diff
@@ -0,0 +1,56 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'WorkSpaces Instance Count',
+    category: 'WorkSpaces',
+    domain: 'Identity Access and Management',
+    description: 'Ensure that the number of Amazon WorkSpaces provisioned in your AWS account has not reached set limit.',
+    more_info: 'In order to manage your WorkSpaces compute resources efficiently and prevent unexpected charges on your AWS bill, monitor and configure limits for the maximum number of WorkSpaces instances provisioned within your AWS account.',
+    recommended_action: 'Ensure that number of WorkSpaces created within your AWS account is within set limit',
+    link: 'https://docs.aws.amazon.com/workspaces/latest/adminguide/workspaces-limits.html',
+    apis: ['WorkSpaces:describeWorkspaces'],
+    settings: {
+        workspace_instance_limit: {
+            name: 'Limit for the number of Workspace instances.',
+            description: 'Plugin results will become aggregated once this value is breached',
+            default: 50
```

Comment:
Can you change the description and default value to be '50' because we always get string from fontend

Reply:
Also regex is missing